src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/TextBox.cs

```diff
@@ -963,8 +965,168 @@ public Typography Typography
             }
         }
 
+        /// <summary>
+        /// Gets or sets displayed <see cref="IconElement"/>.
+        /// </summary>
+        public IconElement Icon
+        {
+            get => (IconElement)GetValue(IconProperty);
+            set => SetValue(IconProperty, value);
+        }
+
+        /// <summary>
+        /// Defines which side the icon should be placed on.
+        /// </summary>
+        public ElementPlacement IconPlacement
+        {
+            get => (ElementPlacement)GetValue(IconPlacementProperty);
+            set => SetValue(IconPlacementProperty, value);
+        }
+
+        public KeyboardNavigation keyboardNavigation { get; set; }
```

Comment:
The property name should be in PascalCase.

Reply:
@zlatanov - Thanks for your suggestion, however we are not using this property, so removed it, these changes have been submitted along with another changes in PR #8604.

Reply:
Closing this PR, you can track it on #8604.